other/appdata/io.github.antimicrox.antimicrox.appdata.xml.in

```diff
@@ -4,15 +4,23 @@
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-3.0+</project_license>
   <name>AntiMicroX</name>
-  <summary>Graphical program used to map keyboard buttons and mouse controls to a gamepad. Useful for playing games with no gamepad support.</summary>
+  <summary>Graphical program used to map gamepad buttons to keyboard, mouse, scripts and macros.</summary>
   <description>
     <p>
-     AntiMicroX is a graphical program used to map keyboard keys and mouse
-     controls to a gamepad. This program is useful for playing PC games using a
-     gamepad that do not have any form of built-in gamepad support. 
+     AntiMicroX is a graphical program used to map gamepad buttons/joysticks
+     to keyboard, mouse, scripts and macros.
+     It can be also used for generating SLD2 configuration (useful for mapping 
+     atypical gamepads to generic ones like xbox360).
     </p>
     <p>
-     AntiMicroX was inspired by QJoyPad but has additional features.
+    It allows mapping of gamepads/joystick buttons to:
```

Comment:
Please use HTML `<ul>` here, too, it's [supported](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-description).